docs/api/cases/cases-api-find-cases.asciidoc

```diff
@@ -0,0 +1,189 @@
+[[cases-api-find-cases]]
+== Find cases API
+++++
+<titleabbrev>Find cases</titleabbrev>
+++++
+
+Retrieves a paginated subset of cases. By default, the first page is returned
+with 20 results per page.
+
+=== Request
+
+`GET <kibana host>:<port>/s/<space_id>/api/cases/_find`
+
+=== Prerequisite
+
+You must have `read` privileges for the *Cases* feature in the *Management*,
+*{observability}*, or *Security* section of the
+<<kibana-feature-privileges,{kib} feature privileges>>, depending on the
+`owner` of the cases you're seeking.
+
+=== Path parameters
+
+`<space_id>`::
+(Optional, string) An identifier for the space. If it is not specified, the
+default space is used.
+
+=== Query parameters
+
+`defaultSearchOperator`::
+(Optional, string) The default operator to use for the `simple_query_string`.
+
+`fields`::
+(Optional, array of strings) The fields in the entity to return in the response.
+
+`owner`::
+(Optional, string or array of strings) A filter to limit the retrieved cases to
+a specific set of applications. Valid values are: `cases`, `observability`,
+and `securitySolution`. If this parameter is omitted, the response contains all
+cases that the user has access to read.
+
+`page`::
+(Optional, integer) The page number to return.
+
+`perPage`::
+(Optional, integer) The number of rules to return per page.
+
+`reporters`::
+(Optional, string or array of strings) Filters the returned cases by the
+reporter's `username`.
+
+`search`::
+(Optional, string) An {es}
+{ref}/query-dsl-simple-query-string-query.html[simple_query_string] query that
+filters the objects in the response.
+
+`searchFields`::
+(Optional, string or array of strings) The fields to perform the
+`simple_query_string` parsed query against.
+
+`sortField`::
```

Comment:
I think we should say that it defaults to `created_at`

Reply:
Done in ed7b3331c93678e255350b47d6e0e31398421e23